### DISCUSSION TITLE
Config issue

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#24375] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

base branches to be picked up with a regex to pick up the following branches:
`rhel-x.y.z`

### What you tried so far.

  "baseBranches": ["/^rhel-[8-10]{1}\\.[0-9]{1}\\.[0-9]{1}/"]

### Relevant debug logs

<details><summary>Logs</summary>

```
Location: renovate.json
Error type: The renovate configuration file contains some invalid settings
Message: Invalid regExp for baseBranches: '/^rhel-[8-10]{1}\.[0-9]{1}\.[0-9]{1}/'```

</details>

**Comment by rarkins** [comment]
Do you have a working pattern on regex101.com?

**Reply by lmilbaum** [reply, marked as answer]
Thanks. Found the problem.
